engine/src/engine.ts

```diff
@@ -1332,7 +1363,10 @@ export default class Engine {
   [Command.ChooseFaction](player: PlayerEnum, faction: string) {
     assert(this.availableCommand.data.includes(faction), `${faction} is not in the available factions`);
     this.setup.push(faction as Faction);
-    this.executeBid(player, faction, 0);
+    if (this.isVersionOrLater("5.0.0")) {
```

Comment:
use `!this.isOnLegacyAuction()` instead

Reply:
Nice catch, I thought that wouldn't work.